### DISCUSSION TITLE
Audio output remains active in sleep mode

### DISCUSSION THREAD

**Discussion by Mike86-CZE** [PE5PVB/TEF6686_ESP32#482] [General]
It has happened to me several times that the TEF went to sleep based on the timer setting and by the next day the battery was completely dead.  I found that after the TEF goes into deep sleep, the audio output is still active (I can hear a noise at max volume).  This further consumes battery power.  Is there a software solution for this or is it a hardware problem and nothing can be done about it?

**Comment by PE5PVB** [comment]
Yes, can't fix that. In stand-by the ESP32, TEF6686 and display are set in sleepmode. The audio amplifier is not digital controlled, so there is no way to set this into standby without hardware changes.
